staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -1425,6 +1429,8 @@ const (
 	MigrationSucceeded VirtualMachineInstanceMigrationPhase = "Succeeded"
 	// The migration failed
 	MigrationFailed VirtualMachineInstanceMigrationPhase = "Failed"
+	// The migration was interrupted and will have to be recovered later
+	MigrationInterrupted VirtualMachineInstanceMigrationPhase = "Interrupted"
```

Comment:
We need to stay away of expanding the phases as it is perceived as finite states and users can rely on them. We need to use conditions if needed